nav2_bt_navigator/README.md

```diff
@@ -75,32 +77,52 @@ returns FAILURE, all nodes are halted and this node returns FAILURE.
 
 The graphical version of this Behavior Tree:
 
-<img src="./doc/simple_parallel.png" title="" width="65%" align="middle">
+<p align="center">
+<img src="./doc/simple_parallel.png" title="" width="40%">
+</p>
 <br/>
 
 The navigate with replanning BT first ticks the `RateController` node which specifies how frequently the `GoalReached` and `ComputePathToPose` should be invoked. Then the `GoalReached` nodes check the distance to the goal to determine if the `ComputePathToPose` should be ticked or not. The `ComputePathToPose` gets the incoming goal pose from the blackboard, computes the path and puts the result back on the blackboard, where `FollowPath` picks it up. Each time a new path is computed, the blackboard gets updated and then `FollowPath` picks up the new goal to compute a control effort for. `controller_id` will specify the type of control effort you'd like to compute such as `FollowPath` `FollowPathSlow` `FollowPathExactly`, etc.
 
 ### Navigate with replanning and simple recovery actions
 
-With the recovery node, simple recoverable navigation with replanning can be implemented by utilizing the [navigate_w_replanning.xml](behavior_trees/navigate_w_replanning.xml) and a sequence of recovery actions. Our custom behavior actions for recovery are:  `clearEntirelyCostmapServiceRequest` for both global and local costmaps and `spin`. A graphical version of this simple recoverable Behavior Tree is depicted in the figure below.
+With the recovery node, simple recoverable navigation with replanning can be implemented by utilizing the [navigate_w_replanning.xml](behavior_trees/navigate_w_replanning.xml) and a sequence of recovery actions. Our custom behavior actions for recovery are:  `clearEntirelyCostmapServiceRequest` for both global and local costmaps, `spin` and `wait`. A graphical version of this simple recoverable Behavior Tree is depicted in the figure below.
 
-<img src="./doc/parallel_w_recovery.png" title="" width="95%" align="middle">
+<p align="center">
+<img src="./doc/parallel_w_recovery.png" title="" width="95%">
+</p>
 <br/>
 
 
 This tree is currently our default tree in the stack and the xml file is located here: [navigate_w_replanning_and_recovery.xml](behavior_trees/navigate_w_replanning_and_recovery.xml).
 
-## Multi-Scope Recoveries
+
+#### Recovery behaviours and condition nodes
```

Comment:
Please explicitly use the word "preemption" in the title and throughout the description. I would add a paragraph at the bottom stating this is the recommended way to implement preemption in the behavior tree following this design pattern.